x-pack/platform/plugins/shared/onechat/public/application/routes.tsx

```diff
@@ -32,6 +34,14 @@ export const OnechatRoutes: React.FC<{}> = () => {
         <OnechatAgentsPage />
       </Route>
 
+      <Route path="/tools/new">
+        <OnechatToolCreatePage />
+      </Route>
+
+      <Route path="/tools/:toolId">
+        <OnechatToolEditPage />
+      </Route>
+
```

Comment:
I just realised we have duplicated /agents route when lookking at this PR, can you fix it ? 🙏 

```
  <Route path="/agents">
        <OnechatAgentsPage />
      </Route>

```